apps/web-giddh/src/app/reports/components/advance-receipt-report/advance-receipt-report.component.html

```diff
@@ -52,6 +48,11 @@
                     <button class="btn btn-filter mr-0" (click)="resetAdvanceSearch()"><span class="icon-cross"></span>
                         {{commonLocaleData?.app_clear_filter}}</button>
                 </div>
+                <div class="advance-icon cursor-pointer btn btn-primary mr-0" tooltip="{{commonLocaleData?.app_advance_search}}"
+                            (click)="openModal()">
+                            <span class="pd-r05">Filter</span>
```

Comment:
Do add multi-lingual labels and avoid hard coded values

Reply:
done